src/read.c

```diff
@@ -708,6 +708,30 @@ static avifResult avifDecoderItemMaxExtent(const avifDecoderItem * item, avifExt
     return AVIF_RESULT_OK;
 }
 
+static avifResult avifDecoderItemValidateAV1(const avifDecoderItem * item)
```

Comment:
Nit/optional: Change "AV1" to "AV1C"?

Reply:
The goal of the function is to validate an item that is backed by an AV1 payload. This function checks more than just the `av1C` block (such as `pixi`).